streaming/base/dataset.py

```diff
@@ -30,6 +32,7 @@
 
 # Time to wait, in seconds.
 TICK = 0.07
+TIMEOUT = 60
```

Comment:
should probably specify which timeout this is

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/147/commits/c231d56705f8d8030f7b2b9dd532040ef6dceafc.